jenkins/mellanox/buildimage-mlnx-all/Jenkinsfile

```diff
@@ -38,11 +38,12 @@ pipeline {
 
 git submodule foreach --recursive '[ -f .git ] && echo "gitdir: $(realpath --relative-to=. $(cut -d" " -f2 .git))" > .git'
 
+CACHE_OPTIONS="SONIC_DPKG_CACHE_METHOD=wcache SONIC_DPKG_CACHE_SOURCE=/nfs/dpkg_cache/mellanox"
 make configure PLATFORM=mellanox
-make SONIC_CONFIG_BUILD_JOBS=1 INSTALL_DEBUG_TOOLS=y target/sonic-mellanox.bin
+make SONIC_CONFIG_BUILD_JOBS=1 INSTALL_DEBUG_TOOLS=y $CACHE_OPTIONS target/sonic-mellanox.bin
 mv target/sonic-mellanox.bin target/sonic-mellanox-dbg.bin
-make SONIC_CONFIG_BUILD_JOBS=1 target/sonic-mellanox.bin
-ENABLE_SYNCD_RPC=y make SONIC_CONFIG_BUILD_JOBS=1 target/docker-syncd-mlnx-rpc.gz target/docker-ptf-mlnx.gz
+make SONIC_CONFIG_BUILD_JOBS=1 $CACHE_OPTIONS target/sonic-mellanox.bin
+ENABLE_SYNCD_RPC=y make SONIC_CONFIG_BUILD_JOBS=1 $CACHE_OPTIONS target/docker-syncd-mlnx-rpc.gz target/docker-ptf-mlnx.gz
```

Comment:
Please be careful about `ENABLE_SYNCD_RPC=y` after merge. I am not sure if the cache will conflicts with non-enabled build.

Reply:
We should have fixed it in PR: https://github.com/Azure/sonic-buildimage/pull/4285/files